pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -348,74 +357,41 @@ type reconstructedVolume struct {
 	volumeSpec          *volumepkg.Spec
 	outerVolumeSpecName string
 	pod                 *v1.Pod
-	pluginIsAttachable  bool
+	attachablePlugin    volumepkg.AttachableVolumePlugin
 	volumeGidValue      string
 	devicePath          string
 	reportedInUse       bool
```

Comment:
We are not using this `reportednUse` variable are we?

Reply:
We do. When reconciler at kubelet tries to VerifyControllerAttachedVolume, it checked whether volume is also reportedInUse in node status to make sure we only try to mount after volume is reported in use (this is checked by reconciler running on master during detach)